### DISCUSSION TITLE
Load/unload hooklift and itrunner

### DISCUSSION THREAD

**Discussion by scrapjo** [Courseplay/courseplay#7669] [Ideas]
Is it possible to have course play running a combine, having the combine empty into a hooklift container(i.e itrunner), then having a truck or a tractor picking this up, emty it and then put it down at the field, and picking up an other container, then repeat?

If unclear i want to have a combine that looks for a emty container(that isnt attached to a truck or trailer) on the field.

Then i want a tractor/ or truck leaving an empty Container on the field, then looking for a full one, picking this up and go and epmty it.

Is this possible?

**Comment by Tensuko** [comment]
No, thats is way to complicated and not much benefit.
It would be cool, yes.
But won't happen in the near feature.
